builtin/credential/ldap/path_config.go

```diff
@@ -200,21 +214,23 @@ func (b *backend) pathConfigWrite(
 }
 
 type ConfigEntry struct {
-	Url          string
-	UserDN       string
-	GroupDN      string
-	UPNDomain    string
-	UserAttr     string
-	Certificate  string
-	InsecureTLS  bool
-	StartTLS     bool
-	BindDN       string
-	BindPassword string
-	DiscoverDN   bool
+	Url           string `json:"url" structs:"url" mapstructure:"url"`
+	UserDN        string `json:"userdn" structs:"userdn" mapstructure:"userdn"`
+	GroupDN       string `json:"groupdn" structs:"groupdn" mapstructure:"groupdn"`
+	UPNDomain     string `json:"upndomain" structs:"upndomain" mapstructure:"upndomain"`
+	UserAttr      string `json:"userattr" structs:"userattr" mapstructure:"userattr"`
+	Certificate   string `json:"certificate" structs:"certificate" mapstructure:"certificate"`
+	InsecureTLS   bool   `json:"insecure_tls" structs:"insecure_tls" mapstructure:"insecure_tls"`
+	StartTLS      bool   `json:"starttls" structs:"starttls" mapstructure:"starttls"`
+	BindDN        string `json:"binddn" structs:"binddn" mapstructure:"binddn"`
+	BindPassword  string `json:"bindpass" structs:"bindpass" mapstructure:"bindpass"`
+	DiscoverDN    bool   `json:"discoverdn" structs:"discoverdn" mapstructure:"discoverdn"`
+	TLSMinVersion uint16 `json:"tls_min_version" structs:"tls_min_version" mapstructure:"tls_min_version"`
```

Comment:
I'd store it as a string and do the mapping later (but verify the value here). Otherwise the user, upon reading the configuration back, will have to figure out what the integer means.